.ci/make-branch-config.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash 
```

Comment:
Perhaps we should have a .ci/scripts directory? Seems we are accumulating a lot of different shell scripts all at the root.

Reply:
That's a good idea. I'll move that in the next iteration, when I have to re-test this anyhow.